platform-sdk/swirlds-state-api/src/main/java/com/swirlds/state/spi/ReadableSingletonState.java

```diff
@@ -39,7 +40,18 @@ public interface ReadableSingletonState<T> {
      *     instance of {@link ReadableKVState}.
      */
     @NonNull
-    String getStateKey();
+    String getStateKey(); // TODO: remove in favor of `getStateId` ?
```

Comment:
`getStateKey()`, `getStateId()`, `getLabel()`, etc. may be extracted to a super-interface, e.g. `ReadableState`

Reply:
Good idea! And I did it. I still propose to get rid of the `getStateKey` method. I checked usages, and it can be substituted with the other methods from `ReadableState` (and it would be better, in my opinion). 

Wdyt?

Reply:
Not relevant anymore